enforcer/src/main/java/org/wso2/micro/gateway/enforcer/server/WebSocketHandler.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
There are couple of places like this. Shall we go through and make them to 2021

Reply:
Yes, will fix license headers to 2021